phpstan.neon.dist

```diff
@@ -234,3 +234,7 @@ parameters:
             message: '#Call to an undefined method TElement of Shopware\\Core\\Framework\\DataAbstractionLayer\\Entity::.*CustomFields.*\(\)#'
             count: 4
             path: src/Core/Framework/DataAbstractionLayer/EntityCollection.php
+
+        - # This attribute is intentionally not final, as it's extended by other field attributes
```

Comment:
i would try to not blow up the config file, but for specific exceptions use inline comments with the error identifier -> that directly explains in the code that this is an exception and it is not just "documented" somewhere else?

What do you think?